spec/app/application.rb

```diff
@@ -34,6 +34,15 @@ class Base < Sinatra::Base
         serializer: FooSerializer
       }
     end
+
+    get '/with-each-serializer/' do
+      Test.create(foo: 'bar1')
+      Test.create(foo: 'bar2')
+
+      json Test.all,
+        root: 'units',
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.